hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/operation/GetJobIdsOperation.java

```diff
@@ -23,28 +23,26 @@
 import com.hazelcast.spi.Operation;
 import com.hazelcast.spi.impl.AllowedDuringPassiveState;
 
-import java.util.ArrayList;
-import java.util.List;
+import static com.hazelcast.jet.impl.util.ExceptionUtil.withTryCatch;
 
 public class GetJobIdsOperation
         extends Operation
```

Comment:
same `AsyncOperation`

also the client message tasks need to be revisited as they're marked as `BlockingMessageTask` currently.